### PR TITLE
Update Customer-Lockbox-Requests.md

### PR DESCRIPTION
**Pull request by panjkov**
Updated step 3 to reflect UX of the new admin center.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/customer-lockbox-requests.md [hunk 1]
```diff
@@ -71,9 +71,9 @@ You can turn on Customer Lockbox controls in the Microsoft 365 admin center. Whe
 
 2. Choose **Settings > Org Settings**.
 
-3. Select **Security & Privacy** > **Customer Lockbox** > **Edit**, and then move the toggle to **On** or **Off** to turn the feature on or off.
+3. Select **Security & Privacy** then click on **Customer Lockbox** in the left column and check the checkbox besides text "Require approval for all data access request" and save changes to turn the feature on.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-lockbox-requests.md]:
```suggestion
3. Select **Security & Privacy**, then select **Customer Lockbox** in the left column. Check the **Require approval for all data access requests** checkbox and save the changes to turn on the feature.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @panjkov, kindly commit my suggested changes if you agree with them. Many thanks.

**Comment by panjkov** [comment]
@JohanFreelancer9 I agree, user can select the entire row.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
Gentle reminder @KCCross kcc

**Pull request merged**